### PR TITLE
Support VRCSDK 3.5.0

### LINKED ISSUE #394: Unity 2022
なんか`Localization/en.po`がクラッシュを引き起こす

### LINKED ISSUE #616: Unity 2022 VRCSDK Support
Planned for 1.6

### PR DESCRIPTION
**Pull request by anatawa12**
Fixes #616 
Fixes #394

### CODE REVIEW

**Comment by anatawa12** [comment]
failing test is expected since the CI is pull_request_target.

File: CHANGELOG-PRERELEASE.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 ### Added
+- Support for VRCSDK 3.5.x `#787`
+  - Actually, previous version of AAO work well in with VRCSDK 3.5.x / Unity 2022 with tiny bugs.
```
Comment (Sayamame-beans) [inline CHANGELOG-PRERELEASE.md:12]:
```suggestion
  - Actually, previous version of AAO works well with VRCSDK 3.5.x / Unity 2022 with tiny bugs.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 ### Added
+- Support for VRCSDK 3.5.x `#787`
+  - Actually, previous version of AAO work well in with VRCSDK 3.5.x / Unity 2022 with tiny bugs.
```
Comment (Sayamame-beans) [inline CHANGELOG.md:12]:
```suggestion
  - Actually, previous version of AAO works well with VRCSDK 3.5.x / Unity 2022 with tiny bugs.
```

File: CHANGELOG-PRERELEASE.md [hunk 3]
```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog].
...
+  - I've fixed some bugs in Unity 2022 in this release.
+  - Since this version, package.json declares Avatar Optimizer is compatible with VRCSDK 3.5.x.
+  - I was planned to release this changes while VRCSDK 3.5.0 is in beta.
+  - However, VRCSDK 3.5.x beta was only 3 hours so I could not.
```
Comment (Sayamame-beans) [inline CHANGELOG-PRERELEASE.md:16]: 3.5.xなのか3.5.0なのかという感じはある

**Review by Sayamame-beans — CHANGES_REQUESTED** [review]

File: CHANGELOG-PRERELEASE.md (hunk 3 above)
Comment (anatawa12) [reply CHANGELOG-PRERELEASE.md:16]: 3.5.0 betaだね
```suggestion
  - However, VRCSDK 3.5.0 beta was only 3 hours so I could not. 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog].
...
+  - I've fixed some bugs in Unity 2022 in this release.
+  - Since this version, package.json declares Avatar Optimizer is compatible with VRCSDK 3.5.x.
+  - I was planned to release this changes while VRCSDK 3.5.0 is in beta.
+  - However, VRCSDK 3.5.x beta was only 3 hours so I could not.
```
Comment (anatawa12) [inline CHANGELOG.md:16]:
```suggestion
  - However, VRCSDK 3.5.0 beta was only 3 hours so I could not. 
```

**Review by Sayamame-beans — APPROVED** [review]
OK!

**Pull request merged**